src/tests/functional/plugin/conformance/subgraphs_dumper/tests/utils/model.cpp

```diff
@@ -40,6 +43,9 @@ TEST_F(ModelUtilsTest, generate_0) {
 }
 
 TEST_F(ModelUtilsTest, generate_1) {
+#if defined(WIN32) && !defined(NDEBUG)
```

Comment:
It can be removed after merge of:
#27174

Reply:
will wait for it then, thanks!